stack/keycloak/migration/keycloak-changelog.yml

```
@@ -24,4 +24,5 @@ includes:
   - path: add-authorities-wahlvorbereitung-unterbrechungsuhrzeit.yml
   - path: add-missing-repository-read-authorities.yml
   - path: add-authorities-wahlvorbereitung-fortsetzungsuhrzeit.yml
-  - path: add-authorities-wahlvorbereitung-eroeffnungsuhrzeit.yml
\ No newline at end of file
+  - path: add-authorities-wahlvorbereitung-urnenwahlschliessungsuhrzeit.yml
+  - path: add-authorities-wahlvorbereitung-eroeffnungsuhrzeit.yml
```

Bitte tauschen da es sonst zu Problemen mit vorhanden Migrationen kommen kann.
```suggestion
  - path: add-authorities-wahlvorbereitung-eroeffnungsuhrzeit.yml
  - path: add-authorities-wahlvorbereitung-urnenwahlschliessungsuhrzeit.yml
```